src/lib/firebase/__mocks__/index.ts

```diff
@@ -0,0 +1,41 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
```

Comment:
I think the license header in `auth-service` still use `Gates Foundation` as well for this part, because it is just an example. Perhaps @eoln can help to clarify whether we need to change to `Mojaloop Foundation` for the example?

Reply:
Btw, for the license, I think I can just mention this in the technical debt and create a follow-up PR to update since it will change almost all files in this repository

Reply:
in master auth-service still lives outdated info -> it should be already fixed in few PR interns are working now -> All newly developed code will be branded by Mojaloop Foundation - Am I right @jgeewax ?